### PR TITLE
3.7.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.7.0 (2019-11-15)
+
+- Added `tag` input to the repeater item component and `orderChange` output to the repeater component to allow tracking user updates to the repeater item order. [#91](https://github.com/blackbaud/skyux-lists/pull/91)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Does this mean the same thing?

"Added the `tag` input and the `orderChange` output to the repeater component to track user updates to the repeater item order."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Fixed the repeater component to no longer throw console errors when a reorderable repeater has no items. [#90](https://github.com/blackbaud/skyux-lists/pull/90) (Thanks [@blackbaud-GavinNicol](https://github.com/blackbaud-GavinNicol)!)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: "Fixed the repeater component to properly handle reorderable repeaters without items."

I could also be swayed to keep the original though but add a comma after "Thanks." :)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Unfortunately, no. `tag` is on the item component, while `orderChange` is on the repeater component.

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: Ah, my bad! I overlooked the word item after `tag`, sorry.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: done!

**Pull request merged**